### PR TITLE
autoconfigbackup - code style fixes

### PR DESCRIPTION
**Pull request by doktornotor**
**N.B. I cannot test this at all without subscription account. Also, there are some clearly whacky cases like intentionally missing closing PHP tags etc. Kindly test yourself before commit. You have been warned.** :stuck_out_tongue_closed_eyes: 

autoconfigbackup.inc
- Update copyright header
- Code style and whitespace fixes
- Use platform_booting()
- Replace backticks with shell_exec()
- Move custom_php_resync_config_command() function from XML to autoconfigbackup.inc and give it an unique prefix
- Also give an unique prefix to custom_php_validation_command()

autoconfigbackup.php
- Fix copyright header
- Code style and whitespace fixes
- Replace deprecated split() with explode(); still one split() occurrence left - cannot be simply replaced with explode() since that changes the output and I have no clue what the code is supposed to do plus no way to test it without subscription account
- Replace backticks with shell_exec()
- Some XHTML fixes

autoconfigbackup.xml
- Move custom_php_resync_config_command() function to autoconfigbackup.inc and give it an unique prefix
- Also give a unique prefix to custom_php_validation_command()
- Fix copyright header
- Fix file permissions (no need for these to be executable)
- Whitespace fixes

autoconfigbackup_backup.php
- Fix copyright header
- Code style and whitespace fixes
- XHTML fixes

autoconfigbackup_stats.php
- Fix copyright header
- Code style and whitespace fixes
- XHTML fixes
- Replace deprecated split() with explode(); still one split() occurrence left - cannot be simply replaced with explode() since that changes the output and I have no clue what the code is supposed to do plus no way to test it without subscription account

crypt_acb.php
- Fix copyright header
- Code style and whitespace fixes
- **N.B.: This code produces "Fatal error: Cannot redeclare stripos()" on any PHP5+ version, really no idea why's it still here for which prehistoric pfS versions. Not touching it otherwise.**

parse_config_upload.inc, parse_config_upload.php, upload_config_filter.php
- Missing/trailing whitespace. All these files are intentionally (653a43f) missing the closing PHP tag for god knows what reason.

### CODE REVIEW

File: config/autoconfigbackup/autoconfigbackup.inc [hunk 1]
```diff
@@ -1,80 +1,100 @@
...
-	
+
 	// We do not need to store this value.
 	unset($_POST['testconnection']);
 }
 
+function acb_custom_php_resync_config_command() {
+	conf_mount_rw();
+	unlink_if_exists("/cf/conf/lastpfSbackup.txt");
+	conf_mount_ro();
```
Comment (rbgarga) [inline config/autoconfigbackup/autoconfigbackup.inc]: While you are here, it would be useful to just call conf_mount_rw/conf_mount_ro if file exists

Comment (doktornotor) [inline config/autoconfigbackup/autoconfigbackup.inc]: Would be even better if unlink_if_exists() handled this on its own... :stuck_out_tongue:

**Comment by cbuechler** [comment]
@doktornotor I added a gold subscription to your portal.pfsense.org account you created in the CLA process, you've certainly put in enough pull requests to get it comped. Thanks! If you try it, please report back. Otherwise I'll merge this soon after testing.

**Comment by doktornotor** [comment]
Thanks, I'll obviously try to test this as well... (Really not doable without the portal access).

**Comment by doktornotor** [comment]
Tweaked the boot check again. Otherwise, seems to work fine here after a quick test.

**Comment by doktornotor** [comment]
Still working after a day, I guess all should be good... One thing I'd like to see fixed are the times being displayed in god knows what timezone, however, really no information how's that handled by the backup server, kinda hard  to even start working on it.

File: config/autoconfigbackup/autoconfigbackup.inc [hunk 2]
```diff
@@ -124,29 +151,30 @@ function test_connection($post) {
...
 	return;
 }
 
 function upload_config($reasonm = "") {
 	global $config, $g, $input_errors;
 
-	/* do nothing when booting */
-	if($g['booting']) 
+	// Do nothing when booting
+	if (platform_booting()) {
```
Comment (rbgarga) [inline config/autoconfigbackup/autoconfigbackup.inc]: I needs to be checked as it was done in https://github.com/pfsense/pfsense-packages/pull/1021/files#diff-a19fa0627590e2e997660ff75d27a301R112

**Comment by doktornotor** [comment]
@cbuechler Might want to bump the package version here, this wasn't done in the PR. Thanks.

**Comment by cbuechler** [comment]
Done, thanks @doktornotor

**Pull request merged**